metricbeat/module/prometheus/collector/_meta/testdata/gauge-with-naninf.plain

```diff
@@ -0,0 +1,7 @@
+# HELP kafka_consumer_records_lag_records The latest lag of the partition
+# TYPE kafka_consumer_records_lag_records gauge
+kafka_consumer_records_lag_records{client_id="consumer1",} NaN
+kafka_consumer_records_lag_records{client_id="consumer2",} +Inf
+kafka_consumer_records_lag_records{client_id="consumer3",} -Inf
+kafka_consumer_records_lag_records{client_id="consumer4",} 5
+
```

Comment:
Could you add here also some examples of the other metric types?

Reply:
yes, I will put them in the same file and rename the file 👍